lib/web_apis/api_importer.es6.js

```diff
@@ -81,21 +122,28 @@ foam.CLASS({
       ],
       code: function(browserName, browserVersion,
         osName, osVersion, apiCatalog) {
-        let interfaceNames = Object.keys(apiCatalog);
-        for (let i = 0; i < interfaceNames.length; i++) {
-          let interfaceName = interfaceNames[i];
-          for (let j = 0; j < apiCatalog[interfaceName].length; j++) {
-            let apiName = apiCatalog[interfaceName][j];
-            this.browserAPIs.put(this.BrowserAPI.create({
-              browserName,
-              browserVersion,
-              osName,
-              osVersion,
-              interfaceName,
-              apiName,
-            }));
+          let browser = this.Browser.create({
+            browserName: browserName,
+            browserVersion: browserVersion,
+            osName: osName,
+            osVersion: osVersion,
+            releaseDate: this.versionHistory
+              .getReleaseDate(browserName, browserVersion),
+          });
+          this.browserDAO.put(browser);
+          let interfaceNames = Object.keys(apiCatalog);
+          for (let i = 0; i < interfaceNames.length; i += 1) {
+            let interfaceName = interfaceNames[i];
+            for (let j = 0; j < apiCatalog[interfaceName].length; j += 1) {
+              let apiName = apiCatalog[interfaceName][j];
+              let webInterface = this.WebInterface.create({
+                interfaceName,
```

Comment:
We should decide whether we are going to use same-key-value-name syntactic sugar. We don't use it above, but we do here. Which shall we go with?